src/pages/settings/InitialSettingsPage.tsx

```diff
@@ -332,13 +332,27 @@ function InitialSettingsPage({currentUserPersonalDetails}: InitialSettingsPagePr
                                 }
                                 iconRight={item.iconRight}
                                 shouldShowRightIcon={item.shouldShowRightIcon}
+                                shouldUseAutoWidth
+                                style={styles.pv4}
                             />
                         );
                     })}
                 </View>
             );
         },
-        [styles.pb4, styles.mh3, styles.sectionTitle, styles.sectionMenuItem, translate, userWallet?.currentBalance, isExecuting, singleExecution, activeCentralPaneRoute, waitForNavigate],
+        [
+            styles.pb4,
+            styles.mh3,
+            styles.pv4,
```

Comment:
I added `styles.pv4` just to fix the lint error. Let me know if we need to tackle this another way, as I can see we're already adding styles to dependencies here.
<img width="1057" alt="Screenshot 2024-10-22 at 18 22 38" src="https://github.com/user-attachments/assets/fe5d2ef7-802c-4f36-9987-3ca3f0ded206">